code/LeftAndMain.php

```diff
@@ -347,7 +348,8 @@ public function getClientConfig()
             // Trim leading/trailing slash to make it easier to concatenate URL
             // and use in routing definitions.
             'name' => $name,
-            'url' => trim($this->Link() ?? '', '/'),
+            'url' => $url,
+            'reactRoutePath' => preg_replace('/^' . preg_quote(AdminRootController::admin_url(), '/') . '/', '', $url),
```

Comment:
This is important because react routes now must be relative to the root route - i.e. if the root route is `/admin` then the asset-admin route must now be `assets` where it used to be `/admin/assets`